packages/redux-query/dist/commonjs/reducers/errors.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
```

Comment:
i don't think the packages/* stuff is necessary. it probably won't hurt tho

Reply:
oh, no, it should not be there, I'll add to .gitignore, this persisted between branches